src/test/java/Pages/UserProfilePage.java

```diff
@@ -0,0 +1,27 @@
+package Pages;
```

Comment:
The prefix of a unique package name is always written in all-lowercase ASCII letters

For example 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/7e56b91d-cc36-4d9b-aedd-dfee6088cfcd)